### PR TITLE
[Twig] Expand the explanation about the humanize filter

### LINKED ISSUE #19758: [Twig Reference] Filter `humanize` add exemple to capitalize behavior
Actually, the doc presents the behavior of replacing underscores with spaces and evoked capitalization of the string. But in the example just before, capitalization is not represented. 

So two options seems possible to me : 

- keep current text and just add other exemple for capitalization behavior : 

> Makes a technical name human readable (i.e. replaces underscores by spaces or transforms camelCase text like `helloWorld` to `hello world` and then capitalizes the string **like `helloWorld` to `Hello World`**).

- group the two behaviors into a single example : 

> Makes a technical name human readable (i.e. replaces underscores by spaces or transforms camelCase text and then capitalizes the string **like `helloWorld` to `Hello World`**).

I'm ready to create the corresponding pull request, I would just like to get advice on the best solution from someone familiar with maintaining the doc.

Personally I find the second solution better.

### PR DESCRIPTION
**Pull request by javiereguiluz**
Fixes #19758.

### CODE REVIEW

File: reference/twig_reference.rst [hunk 1]
```diff
@@ -391,9 +391,18 @@ humanize
 ``text``
     **type**: ``string``
 
-Makes a technical name human readable (i.e. replaces underscores by spaces
-or transforms camelCase text like ``helloWorld`` to ``hello world``
-and then capitalizes the string).
+Transforms the given string into a human readable string (by replacing underscores
+by spaces, capitalizing the string, etc.) It's useful e.g. when displaying
```
Comment (xabbuh) [inline reference/twig_reference.rst]:
```suggestion
with spaces, capitalizing the string, etc.) It's useful e.g. when displaying
```

File: reference/twig_reference.rst [hunk 2]
```diff
@@ -391,9 +391,18 @@ humanize
...
+property/variable names on an interface created for end users:
```
Comment (xabbuh) [inline reference/twig_reference.rst]:
```suggestion
property/variable names to end users:
```

File: reference/twig_reference.rst (hunk 1 above)
Comment (javiereguiluz) [reply reference/twig_reference.rst]: Fixed. Thanks.

File: reference/twig_reference.rst (hunk 2 above)
Comment (javiereguiluz) [reply reference/twig_reference.rst]: Much better. Changed. Thanks.

**Pull request merged**